stm/stm.mli

```diff
@@ -283,9 +285,6 @@ val forward_feedback_hook : (Feedback.feedback -> unit) Hook.t
 
 val get_doc : Feedback.doc_id -> doc
 
-val state_of_id : doc:doc ->
-  Stateid.t -> [ `Valid of Vernacstate.t option | `Expired | `Error of exn ]
```

Comment:
How can clients now retrieve the reason of a missing state? As the new function has no documentation it is hard to tell.

Reply:
It is a bit hard to design the API since we never discussed the interaction model (we should do it in the second UI-related meeting). Can you give scenarios were that knowledge is useful to clients?

Reply:
Well it seems pretty obvious, if a client gets none, what does that mean? Is it pending, was there an error? Anyways hard to tell as the function is not documented.

Reply:
We could rely on other methods of course, but first we need a spec for get_sucess_state